app/services/change_schedule.rb

```diff
@@ -230,8 +230,17 @@ def validate_new_schedule_valid_with_existing_declarations
   end
 
   def applicable_declarations
-    @applicable_declarations ||= participant_profile.participant_declarations
-      .where(state: %w[submitted eligible payable paid])
+    @applicable_declarations ||= begin
```

Comment:
I think this is also used in `validate_cannot_change_cohort_ecf`, which might relax the validation there to allow providers to change cohort for those frozen participants now 🤔

Reply:
Ah yeah you're right. I've pushed an update.